operator/internal/handlers/internal/rules/rules.go

```diff
@@ -118,3 +120,20 @@ func selectRecordingRules(ctx context.Context, k k8s.Client, rs *lokiv1.RulesSpe
 
 	return rl, nil
 }
+
+// OCPAlertManagerEnabled returns true if the Openshift AlertManager is present in the cluster.
+func OCPAlertManagerEnabled(ctx context.Context, opts manifests.Options, k k8s.Client) (bool, error) {
+	if opts.Stack.Tenants != nil && opts.Stack.Tenants.Mode == lokiv1.OpenshiftLogging {
+		var svc corev1.Service
+		key := client.ObjectKey{Name: manifests.MonitoringSVCOperated, Namespace: manifests.MonitoringNS}
+
+		err := k.Get(ctx, key, &svc)
+		if err != nil && !apierrors.IsNotFound(err) {
+			return false, kverrors.Wrap(err, "failed to lookup alertmanager service", "name", key)
+		}
+
+		return err == nil, nil
+	}
+
+	return false, nil
+}
```

Comment:
I suggest we create a separate `openshift` package and place the code in a file `alertmanager.go`, naming the function as `openshift.alertManagerSvcExists`